MAUI/Busy-Indicator/Getting-Started.md

```diff
@@ -56,57 +72,53 @@ namespace BusyIndicatorSample
 
 {% endhighlight %} 
 
-## Create a Simple .NET MAUI Busy Indicator
-
-The [.NET MAUI Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1) control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a [.NET MAUI Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1) (SfBusyIndicator) and configure its elements.
+## Step 4: Add a Basic Busy Indicator
 
-### Adding the .NET MAUI Busy Indicator control
+1. To initialize the control, import the Core namespace into your code.
 
-Step 1: Add the NuGet to the project as discussed in the above reference section. 
+2. Initialize [SfBusyIndicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1)
 
-Step 2: Add the namespace as shown in the following code sample, to initialize the .NET MAUI Busy Indicator.
+Add the namespace as shown in the following code sample.
 
 {% tabs %}
 
 {% highlight xaml %}
 
-    xmlns:core="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
+<ContentPage 
+            ...
+            xmlns:core="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core">
+    <ContentPage.Content> 
+	 	<core:SfBusyIndicator x:Name="busyIndicator"
+                          IsRunning = "True" />
+	</ContentPage.Content> 
+</ContentPage>
 	
 {% endhighlight %}
 
 {% highlight c# %}
 
-    using Syncfusion.Maui.Core;
-
-{% endhighlight %}
-
-{% endtabs %}
-
-Step 3: Set the .NET MAUI Busy Indicator control as the content in the `ContentPage.`
-
-{% tabs %}
-
-{% highlight xaml %}
-
-
-<ContentPage.Content>    
-    <core:SfBusyIndicator x:Name="busyIndicator"
-                          IsRunning = "True" />
-</ContentPage.Content>
-
-
-{% endhighlight %}
+using Syncfusion.Maui.Core;
 
-{% highlight c# %}
-          
-SfBusyIndicator busyIndicator = new SfBusyIndicator(){IsRunning = true}; 
-Content = busyIndicator;  
+namespace BusyIndicatorMauiSample   
```

Comment:
use same namespace in all the code snippets

Reply:
Yes, I have changed.